templates/shared/_charmed-kubeflow-beta-banner.html

```diff
@@ -0,0 +1,9 @@
+<div class="u-fixed-width">
+  <div class="p-notification--information is-inline">
+    <div class="p-notification__content">
+      <h5 class="p-notification__title">Beta-version of  Charmed Kubeflow 1.6 is here:</h5>
+      <p class="p-notification__message">
+        <code>juju deploy kubeflow --channel 1.6/beta</code>. Read more in our <a href="/blog/kubeflow-beta-release-mlops">blog</a> and share your <a href="https://discourse.charmhub.io/t/kubeflow-1-6/6842">feedback</a> with us.</p>
```

Comment:
I think this would benefit from a line break
```suggestion
        <code>juju deploy kubeflow --channel 1.6/beta</code><br>Read more in our <a href="/blog/kubeflow-beta-release-mlops">blog</a> and share your <a href="https://discourse.charmhub.io/t/kubeflow-1-6/6842">feedback</a> with us.</p>
```

Reply:
Done Thanks